src/java.base/share/classes/java/util/ImmutableCollections.java

```diff
@@ -161,7 +161,7 @@ abstract static class AbstractImmutableCollection<E> extends AbstractCollection<
      * Null argument or null elements in the argument will result in NPE.
      *
      * @param <E> the List's element type
-     * @param input the input array
+     * @param coll the input array
```

Comment:
Hello Korov, while at it, can you also change that text to "the input collection". The parameter of the method isn't an array.

Reply:
Hi @jaikiran , the text has been modified.